### PR TITLE
Feature/ui adapter schema on edge

### PR DESCRIPTION
**Pull request by elfenheart**
Schema on edge popovers

schemaonedge

### CODE REVIEW

File: cdap-ui/app/directives/plumb/my-plumb-ctrl.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ angular.module(PKG.name + '.commons')
 
       drawConnections.call(this);
 
-      MyPlumbService.setConnections(this.instance.getConnections());
+      // MyPlumbService.setConnections(this.instance.getConnections());
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/plumb/my-plumb-ctrl.js]: We could probably remove this.

File: cdap-ui/app/index.html [hunk 2]
```diff
@@ -62,6 +62,7 @@
     <script type="text/javascript" src="/assets/bundle/app.js"></script>
     <script type="text/javascript" src="/assets/bundle/tpl.js"></script>
 
-
+    <script type="text/javascript" src="/assets/bundle/app.es6.js"></script>
```
Comment (ajainarayanan) [inline cdap-ui/app/index.html]: We already have this in the other PR right?

**Comment by elfenheart** [comment]
screen shot 2015-08-21 at 11 57 45 am

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**